### DISCUSSION TITLE
質問の「フォロー中のタグ」が機能していない

### DISCUSSION THREAD

**Discussion by fuzz6001** [increments/qiita-discussions#536] [要望・アイデア・不具合]
質問で「フォロー中のタグ」を選択すると、質問が一つも表示されません。いずれのカテゴリでも同じ症状です。

![screenshot 1694247143](https://github.com/increments/qiita-discussions/assets/2080945/c4f15abe-db25-4b41-8e5c-2e0395c58c87)

**Comment by tanaka-asami415** [comment]
@fuzz6001 
ご不便をおかけして申し訳ありません🙇

フィードには1週間以内に投稿された質問のみ表示されるため、その期間に投稿された質問がない可能性がございます。

質問が表示される期間が短いといったご意見 https://github.com/increments/qiita-discussions/discussions/527 もいただいているため
改善を検討しておりますが、直近の開発優先度などから考えると先になってしまいそうです。

そのため、現時点ではお手数ですが検索機能を活用いただけると幸いです🙇
https://help.qiita.com/ja/articles/qiita-search-options

**Comment by fuzz6001** [comment]
>フィードには1週間以内に投稿された質問のみ表示されるため、その期間に投稿された質問がない可能性がございます。

いえ、1週間どころか、今日の投稿すら表示されません。
付いているタグを全てフォローしていないとダメってことはないですよね？

![screenshot 1694430136](https://github.com/increments/qiita-discussions/assets/2080945/762f6372-02d6-42d0-8fdf-ba959cdd3e33)
![screenshot 1694430245](https://github.com/increments/qiita-discussions/assets/2080945/37362ae1-948b-4bfe-9a92-6657e10a5db5)
![screenshot 1694430553](https://github.com/increments/qiita-discussions/assets/2080945/c9249b7e-706a-4568-8bee-b347020f6bae)
![screenshot 1694430562](https://github.com/increments/qiita-discussions/assets/2080945/58a81166-5c5d-404a-910d-97ecb170caee)
![screenshot 1694430628](https://github.com/increments/qiita-discussions/assets/2080945/502439f4-cbba-4998-bfad-3d35ca062907)

**Reply by tanaka-asami415** [reply]
@fuzz6001 
ご確認ありがとうございます🙇

> いえ、1週間どころか、今日の投稿すら表示されません。

もしかすると、質問をフォローされていないのかもしれません。
お手数ですが、記事と質問のタグフォローは別にしていただく必要がございます。

質問のフォローは、タグページより「質問をフォローする」ボタンをクリックすることで行うことができます。
詳細は下記ヘルプをご参照ください🙏
https://help.qiita.com/ja/articles/qiita-question-feed#part-979dfcacbffa3631

**Comment by fuzz6001** [comment]
>もしかすると、質問をフォローされていないのかもしれません。

その通りでした。失礼しました。
質問をフォローしているタグの一覧はどこで見られるでしょうか？

**Reply by tanaka-asami415** [reply]
@fuzz6001 
とんでもございません！原因が分かってよかったです！

> 質問をフォローしているタグの一覧はどこで見られるでしょうか？

大変申し訳ないのですが、現時点では質問をフォローしているタグの一覧を確認できる場所がございません🙇💦
タグフォロー周りの改善項目として検討しております。
